simulator/simulator.py

```diff
@@ -0,0 +1,66 @@
+import firebase_admin
+from firebase_admin import credentials
+from firebase_admin import db
+import datetime
+import time
+import json
+import secrets
+from pprint import pprint
+
+KEY_PATH = './firebase.json'
+COORDINATES_PATH = 'coordinates.json'
+DATABASE_REFERENCE_PATH = 'datas/'
+DATABASE_URL = 'https://smartspace-84906-default-rtdb.firebaseio.com/'
+
+random = secrets.SystemRandom()
+
+cred = credentials.Certificate(KEY_PATH)
+firebase_admin.initialize_app(cred, {
+        'databaseURL': DATABASE_URL
+})
+
+with open(COORDINATES_PATH) as json_file:
+    coordinates_data = json.load(json_file)
+
+used_coordinates_count = 0
+
+def insert_data(data):
+    # timestamp = str(int(time.time() * 1000000))
```

Comment:
Suggest removing this comment since it's not meaningful.

Reply:
addressed in [012318c](https://github.com/ricky9667/smart_space/pull/14/commits/012318ca135c29268f587681ab608a6983f027da).